release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -3338,6 +3338,37 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+//4.16.24
+[id="ocp-4-16-24_{context}"]
+=== RHSA-2024:10147 - {product-title} {product-version}.24 bug fix
+
+Issued: 26 November 2024
+
+{product-title} release {product-version}.24 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:10147[RHSA-2024:10147] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2024:10150[RHBA-2024:10150] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.16.24 --pullspecs
+----
+
+[id="ocp-4-16-24-bug-fixes_{context}"]
+==== Bug fixes
+
+* Previously, if Operator Lifecycle Manager (OLM) was unable to access the secret associated with a service account, OLM would rely on the Kubernetes API server to automatically create a bearer token. With Kubernetes versions 1.22 and later, this action is no longer automatic, so with this release OLM uses the `TokenRequest` API to request a new Kubernetes API token. (link:https://issues.redhat.com/browse/OCPBUGS-44351[*OCPBUGS-44351*])
+
+* Previously, the approval mechanism for certificate signing requests (CSRs) failed because the node name and internal DNS entry for a CSR did not match in terms of character case differences. With this release, an update to the approval mechanism for CSRs skips case-sensitive checks so that a CSR with a matching node name and internal DNS entry does not fail the check because of character case differences. (link:https://issues.redhat.com/browse/OCPBUGS-44629[*OCPBUGS-44629*])
+
+* Previously, HyperShift-based ROKS clusters were unable to authenticate through `oc login`. The web browser displayed an error when it attepted to retrieve the the token after selecting *Display Token*. with this release, `cloud.ibm.com` and other cloud-based endpoints are no longer proxied and authentication is successful. (link:https://issues.redhat.com/browse/OCPBUGS-44277[*OCPBUGS-44277*])
+
+[id="ocp-4-16-24-updating_{context}"]
+==== Updating
+
+To update an existing {product-title} {product-version} cluster to this latest release, see xref:../updating/updating_a_cluster/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster by using the CLI].
+
 //4.16.23
 [id="ocp-4-16-23_{context}"]
 === RHSA-2024:9615 - {product-title} {product-version}.23 bug fix
```

Comment:
If you want to fix this one up while you're in here to match, this is the only other one that is missing the "update" word in the section title:
```suggestion
=== RHSA-2024:9615 - {product-title} {product-version}.23 bug fix
```

Reply:
Oh actually, one addition tweak on this previous entry, too, since it should be a `RHBA`:
```suggestion
=== RHBA-2024:9615 - {product-title} {product-version}.23 bug fix update
```

Reply:
I think it's supposed to be an RHSA actually - I think I fixed it though :)

Reply:
You're right, I was off a section!